sirepo/package_data/template/controls/parameters.py.jinja

```diff
@@ -146,6 +158,15 @@ def update_and_run_simulation(correctors):
     if is_success:
         with open('{{ summaryCSV }}', 'a') as f:
             f.write('{}\n'.format(','.join([str(x) for x in correctors.tolist() + readings])))
+            # POSIT: one table option is enabled.
```

Comment:
Since this is parameters.py.jinja, wouldn't it be better to just generate the code if one table option is enabled and if not, do not generate?

Reply:
[onetable is always true](https://github.com/radiasoft/sirepo/blob/33ac659d1515e9e3311b8d9589cbe9acd61beb00/sirepo/template/madx.py#L874 )

a lot of our code expects that. Eg. all the report generation code that reads from the onetable file. Posit seems sufficient. Do you agree?